src/FakeItEasy/Configuration/FakeConfigurationManager.cs

```diff
@@ -100,12 +102,14 @@ private static Expression BuildArgumentThatMatchesAnything<TValue>()
 
         private static FakeManager GetFakeManagerCallIsMadeOn(ParsedCallExpression parsedCallExpression)
         {
-            if (parsedCallExpression.CallTarget == null)
-            {
-                throw new ArgumentException("The specified call is not made on a fake object.");
-            }
+            return parsedCallExpression.CallTarget == null
+                ? null

```

Comment:
> I have another question though. When `parsedCallExpression.CallTarget == null` we are returning `null` from `GetFakeManagerCallIsMadeOn` which means no exception is thrown from `EnsureCallIsMadeOnAFake`. That implies that `parsedCallExpression.CallTarget == null` means "the calls are made on a fake". Is that correct?

Oh, bother. It is. That's not good, if only from a naming perspective. If the method were `EnsureCallIsNotMadeOnANonFake` it would take the lie away. I don't _love_ it.

> More generally, all calls to `GetFakeManagerCallIsMadeOn` will now receive a null instead of an exception being thrown when `parsedCallExpression.CallTarget == null`. Is that what we want?

I don't mind it so much. Sometime's there's just no fake manager that the call is made on. I think it's safe for the usages we have right now.

I _may_ be able to get rid of `EnsureCallIsMadeOnAFake` and make things look a little bit better. I'll try it out tonight and get back to you.